runtime/gc_stats/CopyForwardStats.hpp

```diff
@@ -72,6 +72,8 @@ class MM_CopyForwardStats : public MM_CopyForwardStatsCore
 	UDATA _doubleMappedArrayletsCandidates; /**< The number of double mapped arraylets that have been visited during marking */
 #endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
 
+	UDATA _startTimeOfCopyForward; /**< The start time of a copy forward cycle */
```

Comment:
CopyForward in the name is redundant -  in class/instance names there is already CopyForward.
But perhaps you can add 'cycle' in the name (i'm guessing, we will in future also have something that is increment specific)!?

Reply:
while touching this file, let's convert UDATA to uintptr_t

Reply:
> i'm guessing, we will in future also have something that is increment specific)!?

Yes, as we'll need to time each increments separately in the future. I'll update it